### PR TITLE
Lisa and Linnéa - project auth

### PR DESCRIPTION
**Pull request by LinneaWa**
frontend: https://lisa-linnea-project-auth.netlify.app/main
backend: https://project-auth-hswv4hjbqq-lz.a.run.app

### CODE REVIEW

**Comment by LinneaWa** [comment]
frontend: https://lisa-linnea-project-auth.netlify.app/main
backend: https://project-auth-hswv4hjbqq-lz.a.run.app

File: backend/server.js [hunk 1]
```diff
@@ -16,9 +18,136 @@ const app = express();
...
+//   Actors: {type: String},
+//   Runtime: {type: String},
+//   Awards: {type: String},
+//   ImdbVotes: {type: String},
+//   ImdbRating: {type: Number},
+//   RottenTomatoScore: {type: String},
+//   Metascore: {type: Number},
+// }); 
+
+// const Movie = mongoose.model("Movie", MovieSchema);
```
Comment (MariaWellander) [inline backend/server.js:125]: Hopefully you get to continue this project and find a way to apply your initial idea. It would be nice if you could create an extra comment to describe why you kept this extra comment below :)

File: frontend/src/GlobalStyles.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components/macro'
```
Comment (MariaWellander) [inline frontend/src/GlobalStyles.js:1]: It's great that you keep practicing the styled components!

File: frontend/src/components/LogIn.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+          dispatch(user.actions.setAccessToken(null))
+          dispatch(user.actions.setError(data.response))
+        })
+      }
+    })
+  }
+
+
+
+
```
Comment (MariaWellander) [inline frontend/src/components/LogIn.js:59]: Maybe you want to just have 1 empty row here instead of 4?

File: backend/server.js [hunk 4]
```diff
@@ -16,9 +18,136 @@ const app = express();
...
+})
+// app.get('/main', async (req, res) =>{
+//   const movies = await Movie.find({});
+//   res.status(200).json({success: true, response: movies});
+// });
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Hello Batman!");
```
Comment (MariaWellander) [inline backend/server.js:150]: Funny theme!

File: frontend/src/components/Register.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector, batch } from 'react-redux'
+import { useNavigate } from 'react-router-dom'
+import { Buttons } from '../GlobalStyles'
+import { InnerWrapper } from '../GlobalStyles';
+import { OuterWrapper } from '../GlobalStyles';
+import { Batman } from '../GlobalStyles';
```
Comment (MariaWellander) [inline frontend/src/components/Register.js:7]: What about instead using comma to import all styles from that file, as you did on row 8? :)

**Review by MariaWellander — COMMENTED** [review]
### Am I able to understand the code easily? Does the code have lots of duplicates? Could it be tidied up?

We really like the code. It's simple, which is needed for new learners like us. Amazing describing comments! Maybe tidy it up a bit and delete extra rows and extra import rows that could be added to the same row (cause it's from the same file)? What do you think about that :)

### Does it work or does anything look broken?

We've closely tried it in and out and everything works - super :)

### Does it follow the general and blue requirements?
Yes, it does!

### To sum it up

Very good work, Linnéa and Lisa! We think that despite the fact that you said it was an difficult week, you guys managed to deliver an complete product using both backend and frontend. Very classy styling. We understand that you wanted to use your own dataset. Even though you couldn't do it now, we believe that you can find a solution later :) Wonderful work this week! By proud of yourselves!

Reviewed by:
//Kartika and Maria <3
